modules/nf-core/stitch/meta.yml

```diff
@@ -0,0 +1,146 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "stitch"
+description: "STITCH is an R program for reference panel free, read aware, low coverage sequencing genotype imputation. STITCH runs on a set of samples with sequencing reads in BAM format, as well as a list of positions to genotype, and outputs imputed genotypes in VCF format."
+keywords:
+  - imputation
+  - genomics
+  - vcf
+  - bgen
+  - cram
+  - bam
+  - sam
+tools:
+  - "stitch":
+      description: "STITCH - Sequencing To Imputation Through Constructing Haplotypes"
+      homepage: "https://github.com/rwdavies/stitch"
+      documentation: "https://github.com/rwdavies/stitch"
+      tool_dev_url: "https://github.com/rwdavies/stitch"
+      doi: "10.1038/ng.3594"
+      licence: "['GPL v3']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing information about the set of positions to run the imputation over
+        e.g. `[ id:'test' ]`
+
+  - meta3:
+      type: map
+      description: |
+        Groovy Map containing information about the reference genome used
+        e.g. `[ id:'test' ]`
+
+  - posfile:
+      type: file
+      description: |
+        Tab-separated file describing the variable positions to be used for imputation. Refer to the documentation for the `--posfile` argument of STITCH for more information.
+      pattern: "*.tsv"
+
+  - input:
+      type: directory
+      description: |
+        Folder of pre-generated input RData objects used when STITCH is called with the `--regenerateInput FALSE` flag. It is generated by running STITCH with the `--generateInputOnly TRUE` flag.
+      pattern: "input"
+
+  - rdata:
+      type: directory
+      description: |
+        Folder of pre-generated input RData objects used when STITCH is called with the `--regenerateInput FALSE` flag. It is generated by running STITCH with the `--generateInputOnly TRUE` flag.
+      pattern: "RData_in"
+
+  - chromosome_name:
+      type: string
+      description: Name of the chromosome to impute. Should match a chromosome name in the reference genome.
+
+  - K:
+      type: integer
+      description: Number of ancestral haplotypes to use for imputation. Refer to the documentation for the `--K` argument of STITCH for more information.
+
+  - nGen:
+      type: integer
+      description: Number of generations since founding of the population to use for imputation. Refer to the documentation for the `--nGen` argument of STITCH for more information.
+
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing information about the set of samples
+        e.g. `[ id:'test' ]`
+
+  - collected_crams:
+      type: file
+      description: List of sorted BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+
+  - collected_crais:
+      type: file
+      description: List of BAM/CRAM/SAM index files
+      pattern: "*.{bai,crai,sai}"
+
+  - cramlist:
```

Comment:
The input could be crams or bams. Is there a particular reason why you use crams only?

Reply:
Good point will implement this. Just for the module I need to use either cram or bam for the name of the input, even though it can accept both